modules/mobile/spec/request/observations_request_spec.rb

```diff
@@ -0,0 +1,73 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require 'support/mr_client_helpers'
```

Comment:
Now that you've removed that stub, you no longer need to import the client or the client helper or include the client helper on line 9 below. That also applies to the other spec file.